app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,8 @@
   </head>
 
   <body>
+      <p class="notice"><%= notice %></p>
+      <p class="alert"><%= alert %></p>
     <%= yield %>
```

Comment:
- [ ] [OPTIONAL] It would be nice if you allow the user to log out from the web page on your project